lib/yardstick/v2_client/user_exam.rb

```diff
@@ -5,7 +5,7 @@ module Yardstick
   module V2Client
     class UserExam
       include RemoteModel
-      attr_accessor :token, :id, :paths, :user, :user_id, :exam_id, :exam_form_id, :finished_at
+      attr_accessor :token, :id, :paths, :user, :user_id, :exam_id, :exam_form_id, :finished_at, :locale, :marking_status
```

Comment:
Why is marking status here? Are we rendering that back from the API now?

Reply:
i misunderstood the use of `marking_status`. I thought it was needed to query the Measure API